alf/environments/suite_procgen.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2021 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gym3
+from procgen import ProcgenGym3Env
+
+from alf.environments.alf_gym3_wrapper import AlfGym3Wrapper
+
+
+def _load_procgen(env_name: str, batch_size: int = 1, render: bool = False):
+    env = ProcgenGym3Env(
+        num=batch_size,
+        env_name=env_name,
+        render_mode='rgb_array' if render else None)
+    # This extracts the 'rgb' field in the observation dictionary
+    # every time when step() is called, so that the observation is
+    # an image that our algorithm can consume.
+    env = gym3.ExtractDictObWrapper(env, 'rgb')
```

Comment:
Does the original observation have other field other than 'rgb'?

Reply:
Yes for `procgen` there is only one field `rgb`.